user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java

```diff
@@ -88,6 +96,7 @@ public void onWindowClosing( ClosingEvent event ) {
       }
     } );
 
+    setIsAsyncModeEnabled();
```

Comment:
Thank you Thomas for review. Your comments are very valuable. The server calling once when MantleTabPanel created on home page. Next config request to server is sent from pentaho-reporting-plugin.